doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -290,6 +290,13 @@ nRF Desktop
     The PDK can act as a sample mouse or keyboard.
     It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
     The PDK uses MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) and supports firmware updates using the :ref:`nrf_desktop_dfu`.
+  * Support for the nRF54H20 DK with the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
```

Comment:
You should also add a record for nRF54H DK under `nRF Desktop: Board configuration`

Reply:
Don't understand you.